recipe_scrapers/_schemaorg.py

```diff
@@ -265,7 +265,11 @@ def _extract_howto_instructions_text(self, schema_item):
         return instructions_gist
 
     def instructions(self):
-        instructions = self.data.get("recipeInstructions") or ""
+        instructions = (
+            self.data.get("recipeInstructions")
+            or self.data.get("RecipeInstructions")
+            or ""
+        )
```

Comment:
Instead of modifying the base `schema.org` scraper, I'd prefer if we could add an exception for this individual recipe website, using `_schema_cls` from #1365.